includes/Documents/OrderDocument.php

```diff
@@ -1808,6 +1808,18 @@ public function get_due_date(): int {
 			return 0;
 		}
 
+		return $this->calculate_due_date( $due_date_days, $this->order );
```

Comment:
Is it necessary to pass the order as a parameter, considering it’s always accessible via the document object?

Reply:
Good point. Fixed.

Reply:
Remember to update here: https://github.com/wpovernight/woocommerce-pdf-ips-templates/pull/340